src/pages/ProfilesExplorerView/domain/variables/FiltersVariable/filters-ops.ts

```diff
@@ -0,0 +1,120 @@
+import { AdHocVariableFilter } from '@grafana/data';
+import { isRegexOperator } from '@shared/components/QueryBuilder/domain/helpers/isRegexOperator';
+import { CompleteFilter, OperatorKind } from '@shared/components/QueryBuilder/domain/types';
+
+export const convertPyroscopeToVariableFilter = (filter: CompleteFilter): AdHocVariableFilter => {
```

Comment:
Because the `.tsx` extension has been changed, this file appears as new. But what this PR did is to keep this function and to add the 3 new functions below.